whois-api/src/main/java/net/ripe/db/whois/api/mail/dequeue/MessageParser.java

```diff
@@ -328,4 +353,13 @@ void addCredential(final Credential credential) {
             credentials.add(credential);
         }
     }
+
+    private String decodeBase64Encoding(final String paragraph) {
+        try {
+            return new String(new Base64().decode(paragraph.getBytes()));
+        } catch (Exception e) {
+            LOGGER.info("failed to decode base64 for ");
```

Comment:
Missing ```e.getClass().getName()``` and ```e.getMessage()``` from LOGGER statement?

Reply:
Yes thanks for pointing it out, I somehow missed it